### DISCUSSION TITLE
Add possibility to use Azure CosmosDb with ServiceStack OrmLite

### DISCUSSION THREAD

**Discussion by vgotra** [ServiceStack/Discuss#74] [Ideas]
As CosmosDb is free service for MS Azure, it can be useful to have a possibility to use it with ServiceStack OrmLite

**Comment by mythz** [comment]
As CosmosDb isn't an RDBMS and there's no ADO.NET provider for it, OrmLite wont be able to support it.

Best we could do is create a typed library for it like we do for [PocoDynamo for DynamoDb](https://docs.servicestack.net/aws-pocodynamo), but given there's already a decent .NET client for CosmosDB, creating and maintaining a new client wouldn't be a good use of our limited time.
